docs/kubernetes/beginners-guide-to-kubernetes-part-3-objects/index.md

```diff
@@ -60,7 +60,7 @@ Each manifest has four necessary parts:
 - Metadata about the resource
 - Though not required by all objects, a spec which describes the desired behavior of the resource is necessary for most objects and controllers.
 
-In the case of this example, the API in use is `v1`, and the `kind` is a Pod. The metadata field is used for applying a name, labels, and annotations. Names are used to differentiate resources, while labels are used to group like resources. Labels will come into play more when defining [Services](#services) and [Deployments](/docs/applications/containers/kubernetes/beginners-guide-to-kubernetes-controllers/#deployments). Annotations are for attaching arbitrary data to the resource.
+In the case of this example, the API in use is `v1`, and the `kind` is a Pod. The metadata field is used for applying a name, labels, and annotations. Names are used to differentiate resources, while labels are used to group like resources. Labels will come into play more when defining [Services](#services) and [Deployments](/docs/kubernetes/beginners-guide-to-kubernetes-controllers/#deployments). Annotations are for attaching arbitrary data to the resource.
```

Comment:
For this first pass, some links won't work because they were still using old slugs. For example:

- Part 4 of Beginners Kubernetes is at `/docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/`
- Part 3 links to the Deployments section by using `/docs/applications/containers/kubernetes/beginners-guide-to-kubernetes-controllers/#deployments`, which is an older alias for Part 4
- This PR changes that to `/docs/kubernetes/beginners-guide-to-kubernetes-controllers/#deployments`
- This alias doesn't exist on the guide